free-programming-books-ko.md

```diff
@@ -30,7 +31,7 @@
 
 * [AN INTRODUCTION TO PROGRAMMING IN GO 한글 번역](http://www.codingnuri.com/golang-book/index.html)
 * [Go Tour 한글 번역](http://go-tour-kr.appspot.com)
-
+* [가장 빨리 만나는 Go 언어](http://www.pyrasis.com/private/2015/06/01/publish-go-for-the-really-impatient-book)
```

Comment:
Is this book really free? I tried reading a chapter but got redirected to a website selling the book.